packages/ilmomasiina-components/src/api.ts

```diff
@@ -43,15 +42,12 @@ export function configureApi(url: string) {
   apiUrl = url;
 }
 
-export default async function apiFetch(uri: string, {
-  method = 'GET', body, headers, accessToken, signal,
+export default async function apiFetch<T = any>(uri: string, {
```

Comment:
`any` spreads like a virus - avoid (fix errors from this if necessary)
```suggestion
export default async function apiFetch<T = unknown>(uri: string, {
```

Reply:
The previous implementation does the same thing but even worse, as it didn't even consider the response being undefined. that `T = any` is just a backwards compatability layer.